change/@fluentui-react-drawer-a264b8f6-8d3e-44c6-9c17-b5ace7f83e38.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "refactor: rename main DrawerOverlay/DrawerInline components",
```

this is breaking change, lets make it obvious

in react-components/unstable its not as we are doing the rename re-exports

```suggestion
  "comment": "BREAKING CHANGE: rename main DrawerOverlay/DrawerInline components",
```